cmd-info.c

```diff
@@ -680,13 +681,52 @@ static int read_arg_spec(void *arg)
 	return 0;
 }
 
+static int fill_record_date(void *arg)
+{
+	struct fill_handler_arg *fha = arg;
+	time_t current_time;
+
+	time(&current_time);
+
+	dprintf(fha->fd, "record_date:%s", ctime(&current_time));
+	dprintf(fha->fd, "elapsed_time:%s\n", fha->elapsed_time);
+	return 0;
+}
+
+static int read_record_date(void *arg)
+{
+	struct ftrace_file_handle *handle = arg;
+	struct uftrace_info *info = &handle->info;
+	char buf[4096];
+
+	if (fgets(buf, sizeof(buf), handle->fp) == NULL)
+		return -1;
+
+	if (strncmp(buf, "record_date:", 12))
+		return -1;
+
+	info->record_date = copy_info_str(&buf[12]);
+
+	if (fgets(buf, sizeof(buf), handle->fp) == NULL)
+		return -1;
+
+	if (strncmp(buf, "elapsed_time:", 13))
+		return -1;
+
+	sprintf(buf, "%s\n# %-20s: %s", info->record_date, "elapsed time",
+		copy_info_str(&buf[13]));
```

Comment:
@namhyung , hum.. I thought it'd be better to has two items in uftrace_info (i.e. record_date and elapsed_time).
What do you think about it ?

Reply:
Yes, I think it's good to have uftrace_info->elapsed_time member.  What I was asking is not to add an item to `enum uftrace_info_bits`.

Reply:
Ok !